src/foremast/s3/s3apps.py

```diff
@@ -67,7 +71,11 @@ def create_bucket(self):
                 LOG.error("Shared bucket %s does not exist", self.bucket)
                 raise S3SharedBucketNotFound
         else:
-            _response = self.s3client.create_bucket(ACL=self.s3props['bucket_acl'], Bucket=self.bucket)
+            if self.region == 'us-east-1':
+                _response = self.s3client.create_bucket(ACL=self.s3props['bucket_acl'], Bucket=self.bucket)
+            else:
+                _response = self.s3client.create_bucket(ACL=self.s3props['bucket_acl'], Bucket=self.bucket,
+                                                        CreateBucketConfiguration={'LocationConstraint': self.region})
```

Comment:
Is there an issue if we pass LocationConstraint for any region?

Reply:
lol yeah, "us-east-1" does not work in there. All other regions are valid, but it needs to exclude for the "standard" region.